core/templates/dev/head/css/oppia.css

```diff
@@ -2988,3 +2988,333 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+
+/*
+  about / learners / creators pages
+*/
+.clearfix {
+  display: table;
+  content: "";
+  clear: both;
+}
+
+.about-header {
+  background-color: rgb(54, 222, 207);
+  background-image: url("../images/general/icons-bg.png");
+  background-blend-mode: soft-light;
+  color: #fff;
+  text-align: center;
+  padding: 10%;
+}
+
+.about-card {
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.about-content {
+  width: 60%;
+  padding: 0;
+}
+
+.about-extra-container {
+  width: 100%;
+}
+
+.about-extra-content {
+  width: 60%;
+  margin: 0 auto;
+  position: relative;
+}
+
+.about-otter-container {
+  position: absolute;
+  display: inline-block;
+  bottom: -2.3em;
+}
+
+.otter-speech-card {
+  width: 50%;
+  float: right;
+}
+
+.otter-speech-bubble {
+  position: relative;
+  padding: 50px 15px;
+  margin: 1em 0 3em;
+  text-align: center;
+  background: #fff;
+  border-radius: 5px;
+  border-left: none;
+}
+
+.otter-speech-bubble:before {
+  content: "";
+  position: absolute;
+  z-index: 10;
+  bottom: -1.7em;
+  left: 1em;
+  width: 50px;
+  height: 30px;
+  background: #fff;
+  display: block;
+  border-bottom-right-radius: 80px 50px;
+}
+
+.otter-speech-bubble:after {
+  content: "";
+  position: absolute;
+  z-index: 10;
+  bottom: -1.7em;
+  left: 1em;
+  width: 20px;
+  height: 30px;
+  background: #eee;
+  border-bottom-right-radius: 40px 50px;
+  display: block;
+}
+
+.about-header h1 {
+  color: #fff;
+  font-weight: bold;
+  font-size: 3em;
+  text-shadow: 1px 2px 3px #38b1a5;
+  font-family: "Capriola", "Roboto", Arial, sans-serif;
+}
+
+.about-extra-container h1,
```

Comment:
Do we need _all_ of these?

Also, if there are any other unused styles, could you please remove them too?